site/specs-temp/numbers.json

```diff
@@ -17816,7 +17819,260 @@
         "required": [
           "eventType"
         ]
+      },
+      "AccountResponse": {
+        "type": "object",
+        "properties": {
+          "Account": {
+            "$ref": "#/components/schemas/Account"
+          },
+          "AddressErrorDescription": {
+            "$ref": "#/components/schemas/AddressErrorDescription"
+          }
+        }
+      },
+      "Address" : {
+        "type" : "object",
+        "properties" : {
+          "HousePrefix" : {
+            "type" : "string"
+          },
+          "HouseNumber" : {
+            "type" : "string"
+          },
+          "HouseSuffix" : {
+            "type" : "string"
+          },
+          "PreDirectional" : {
+            "type" : "string"
+          },
+          "StreetName" : {
+            "type" : "string"
+          },
+          "StreetSuffix" : {
+            "type" : "string"
+          },
+          "PostDirectional" : {
+            "type" : "string"
+          },
+          "AddressLine2" : {
+            "type" : "string"
+          },
+          "City" : {
+            "type" : "string"
+          },
+          "RegionId" : {
+            "type" : "integer",
+            "format" : "int32",
+            "deprecated": true
+          },
+          "StateCode" : {
+            "type" : "string"
+          },
+          "Zip" : {
+            "type" : "string"
+          },
+          "PlusFour" : {
+            "type" : "string"
+          },
+          "County" : {
+            "type" : "string"
+          },
+          "Country" : {
+            "type" : "string"
+          },
+          "LegacyAddressId" : {
```

Comment:
Do we want this legacy parameter defined? 

we probably need a description explaining what this is

Reply:
I can remove this field from the schema, I don't know do we need it.